keepercommander/api.py

```diff
@@ -1691,7 +1692,7 @@ def fix_data(d):
 
     request = {
         'command': 'get_enterprise_data',
-        'include': ['managed_companies', 'licenses', 'nodes']
+        'include': ['managed_companies']
     }
```

Comment:
MSP is the enterprise that supports an additional data type: managed_companies
Do we really nned to separate it the the separate method?